src/vorta/tray_menu.py

```diff
@@ -70,6 +72,13 @@ def set_tray_icon(self, active=False):
         """
         Use white tray icon, when on Gnome or in dark mode. Otherwise use dark icon.
         """
+        last_backup = EventLogModel.select().where(
+            EventLogModel.subcommand == 'create' & EventLogModel.returncode == '1'
+            & EventLogModel.repo_url != '(NULL)').select(pw.fn.MAX(EventLogModel.start_time)).scalar()
+        last_backup_formatted = last_backup.strftime('%d %B %H:%M') if last_backup else self.tr("Never")
         icon_name = f"icons/hdd-o{'-active' if active else ''}.png"
+        self.setToolTip(self.tr("Vorta\nStatus: Running") if active else
+                        self.tr("Vorta\nStatus: Idle\nLast Backup: %s")
+                        % last_backup_formatted + self.tr("\nNext Backup: %s" % self.app.scheduler.next_job))
```

Comment:
```suggestion
        self.setToolTip(
            self.tr("Vorta\nStatus: Running") if active else self.tr(
                "Vorta\nStatus: Idle\nLast Backup: %s\nNext Backup: %s" %
                (last_backup_formatted, self.app.scheduler.next_job)))
```
Minor formatting thing, since you can use tuples with placeholders.